File: API/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.AsyncTask;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import java.io.InputStream;
import java.io.InputStreamReader;
import java.net.HttpURLConnection;
import java.net.URL;

public class MainActivity extends AppCompatActivity {

    TextView output;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        output = findViewById(R.id.Output);
    }

}
public class MyTask extends AsyncTask<String, Void, String> {

    TextView output;


    public Mytask(TextView output){
        this.output = output;
    }

    @Override protected String doInBackground(String... strings) {
        String stringURL = strings[0]) ;
        InputStream inputStream = null;
        InputStreamReader inputStreamReader = null;
        StringBuffer buffer = null;

        try {
            URL url = new URL(stringURL);
            HttpURLConnection connection = (HttpURLConnection) url;
        }
        return null;
    }

    @Override
    protected void onPreExecute(){
        super.onPreExecute();
    }

    @Override
    protected void onPostExecute(String s){
        super.onPostExecute(s);
    }


}

File: TipCalc2/app/src/main/java/com/example/tipcalc/MainActivity.kt
package com.example.tipcalc

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

public class MainActivity extends AppbarActivity {

        private double currentBillTotal;
        private int currentCustomPercent;
        private EditText top10EditText;
        private EditText top15EditText;
        private EditText top20EditText;
        private EditText total10EditText;
        private EditText total15EditText;
        private EditText total20EditText;
        private TextView customTipTextView;
        private EditText billEditText;
        private EditText tipCustomEditText;
        private EditText totalCustomEditText;
    }

    @Override
        protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main)
    }

}


